include/openenclave/internal/trace.h

```diff
@@ -32,6 +37,12 @@ void initialize_log_config(void);
         oe_log(level, __VA_ARGS__); \
     } while (0)
 
+#define OE_TRACE_DUMP(level, ...)        \
```

Comment:
why are we defining this? Is it necessary? Can one of the existing functions cover it?